frontend/util.lua

```diff
@@ -350,6 +350,22 @@ function util.arrayReverse(t)
     end
 end
 
+-- Test whether t contains a value equal to v
```

Comment:
```suggestion
--- Test whether t contains a value equal to v
```

Also for arrayReverse above.